source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -468,7 +504,64 @@ void HttpConnectionManagerConfig::processFilter(
   bool is_terminal = factory.isTerminalFilter();
   Config::Utility::validateTerminalFilters(proto_config.name(), factory.name(), filter_chain_type,
                                            is_terminal, last_filter_in_current_config);
-  filter_factories.push_back(callback);
+  auto filter_config_provider = filter_config_provider_manager_.createStaticFilterConfigProvider(
+      callback, proto_config.name());
+  ENVOY_LOG(debug, "      name: {}", filter_config_provider->name());
+  ENVOY_LOG(debug, "    config: {}",
+            MessageUtil::getJsonStringFromMessage(
+                proto_config.has_typed_config()
+                    ? static_cast<const Protobuf::Message&>(proto_config.typed_config())
+                    : static_cast<const Protobuf::Message&>(
+                          proto_config.hidden_envoy_deprecated_config()),
+                true));
+  filter_factories.push_back(std::move(filter_config_provider));
+}
+
+void HttpConnectionManagerConfig::processDynamicFilterConfig(
+    const std::string& name, const envoy::config::core::v3::ExtensionConfigSource& config_discovery,
+    FilterFactoriesList& filter_factories, const char* filter_chain_type,
+    bool last_filter_in_current_config) {
+  ENVOY_LOG(debug, "      name: {}", name);
+  ENVOY_LOG(debug, " discovery: {}", MessageUtil::getJsonStringFromMessage(config_discovery));
```

Comment:
Might this be an issue if we have credentials in `ConfigSource`? There is some redaction support if you want to keep this behavior.